### PR TITLE
more rubocop

### PR DESCRIPTION
**Pull request by grosser**
@sandlerr it never stops ...

### CODE REVIEW

File: config/initializers/app_routes.rb [hunk 1]
```diff
@@ -2,6 +2,6 @@ module AppRoutes
   # memoize this due to a bug in MRI that shows up in multi-threaded code
   # see https://github.com/puma/puma/issues/647, search for "bug in MRI"
```
Comment (sandlerr) [inline config/initializers/app_routes.rb]: the linked bug is: https://bugs.ruby-lang.org/issues/10871 seems closed in ruby 2.2

**Comment by sandlerr** [comment]
what do you think of just excluding all old migrations from rubocop?

**Comment by sandlerr** [comment]
:+1:

File: config/initializers/app_routes.rb (hunk 1 above)
Comment (grosser) [inline config/initializers/app_routes.rb]: let's do a separate PR for that ...

**Comment by grosser** [comment]
too late on migrations ... hard to exclude all old without all listing them 1-by-1 as exclude ... so I'd rather fix them up
